science/paraview/Portfile

```diff
@@ -123,7 +120,8 @@ pre-configure {
                -DMPI_C_COMPILER=${mpi.cc} \
                -DMPI_CXX_COMPILER=${mpi.cxx} \
                -DMPIEXEC=${mpi.exec} \
-               -DMPI_Fortran_COMPILER=${mpi.fc}
+               -DMPI_Fortran_COMPILER=${mpi.fc} \
+               -DVTK_USE_SYSTEM_DIY2:BOOL=OFF
```

Comment:
I ran into a problem on my Sierra machine. I think this is specific to mpi, so I put this here:
```
           # resolves a problem on Sierra:
           # /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/__threading_support:156:1: error: unknown type name 'mach_port_t'
           # mach_port_t __libcpp_thread_get_port();
           # https://github.com/arvidn/libtorrent/issues/2364#issuecomment-336175406
           configure.cxxflags-append \
               -D_DARWIN_C_SOURCE
```

Reply:
I'll try to put a conditional on Sierra to add this configuration argument.